### DISCUSSION TITLE
CozyRooms (11/09/2023)

### DISCUSSION THREAD

**Discussion by progettistamatto** [sapienzastudentsnetwork/basi-di-dati-2#47] [Esami]
In questa discussion a tema il progetto "CozyRooms", oggetto del testo di esame del 11 settembre 2023, discuteremo di eventuali proposte di soluzione, dubbi o scelte progettuali in merito alla realizzazione del progetto in questione

**Comment by Obrigad0** [comment]
[PDF esame](https://github.com/sapienzastudentsnetwork/bdii2223/files/12832806/esameBD2settembre11anno2023.pdf)
Condivido con voi il mio esame con risultato di 30L, spero che possa esservi d'aiuto per superare il corso <3
(ps. i segni rossi sono delle correzioni lasciate dal professore)

**Comment by progettistamatto** [comment]
Per questo progetto è disponibile [[QUI]](https://github.com/Exyss/university-notes/raw/main/Bachelor/Secondo%20Anno/Basi%20di%20Dati%20II/CozyRooms.pdf) anche una soluzione proposta da Simone Bianco (Exyss), che dichiara essere il suo esame da 30L

> [!Warning]
_Le soluzioni di Simone Bianco fanno uso degli **Entity-Relationship (ER)** Diagrams su cui il corso si basava fino all'Anno Accademico 2023/2024, mentre **attualmente vengono utilizzati gli UML Class** Diagram. Sebbene i due siano estremamente simili, vi sono **alcune discordanze tra di loro**, in particolare l'**inversione di posizione dei vincoli di molteplicità agli estremi delle relationship**. Nonostante ciò, il rimanente delle fasi di analisi e progettazione delle soluzioni proposte risulta inaffetto da tale cambiamento. Per tanto, una volta comprese le differenze tra ER e Classi UML, le soluzioni risultano normalmente fruibili._

Rispondi a questo commento taggando @Exyss per eventuali dubbi o segnalazioni in merito

**Reply by Besugo03** [reply]
@Exyss una domanda, ma il livello di affidabilitá non dovrebbe essere associato ad un utente, piuttosto che ad una struttura? Non avrebbe piú senso essere a conoscenza che lo sconto sia volto ad attrarre utenti con livelli di affidabilitá piú alti, in modo che il gestore della struttura possa avere clienti piú affidabili?

**Comment by progettistamatto** [comment]
Per questo progetto è disponibile [[QUI]](https://github.com/SimoneLid/BD2/raw/main/Prove%20D'Esame/2023-9-11-CozyRooms/CozyRooms.pdf) una soluzione proposta da Simone Lidonnici (@SimoneLid)

**Comment by CosaFaSelect1** [comment]
Per questo progetto è disponibile [[QUI]](https://github.com/CasuFrost/university-notes-Informatica/raw/main/Secondo%20Anno/Secondo%20Semestre/Basi%20di%20Dati%202/altra%20roba/CozyRooms.pdf) una soluzione proposta da Marco Casu
